velox/functions/prestosql/registration/DateTimeFunctionsRegistration.cpp

```diff
@@ -89,6 +89,8 @@ void registerSimpleFunctions(const std::string& prefix) {
       {prefix + "to_unixtime"});
 
   registerFromUnixtime(prefix + "from_unixtime");
+  registerFunction<CurrentTimezoneFunction, Varchar>({"current_timezone"});
```

Comment:
```suggestion
  registerFunction<CurrentTimezoneFunction, Varchar>({prefix + "current_timezone"});
```
shouldn't be prefix?

Reply:
Yes. I missed it. Thanks.